vendor/BUILD

```diff
@@ -41,8 +41,6 @@ filegroup(
         "//vendor/github.com/aws/aws-sdk-go/service/elb:all-srcs",
         "//vendor/github.com/aws/aws-sdk-go/service/kms:all-srcs",
         "//vendor/github.com/aws/aws-sdk-go/service/sts:all-srcs",
-        "//vendor/github.com/aws/aws-sdk-go/vendor/github.com/go-ini/ini:all-srcs",
```

Comment:
Seems unrelated.

Reply:
I know but:
1. I don't control what `godep` does :-)
2. Less vendored code is better